src/PlanetParameters.jl

```diff
@@ -47,4 +47,5 @@ Planet.TSI(ps::AbstractEarthParameterSet)            = 1362
 Planet.MSLP(ps::AbstractEarthParameterSet)           = 1.01325e5
 Planet.T_surf_ref(ps::AbstractEarthParameterSet)     = 290.0
 Planet.ΔTv_ref(ps::AbstractEarthParameterSet)        = 60.0
```

Comment:
```suggestion
```

Reply:
I've modified this for now to
```julia
Planet.ΔTv_ref(ps::AbstractEarthParameterSet)        = Planet.T_surf_ref(ps) - Planet.T_min_ref(ps)
```
Waiting to hear back regarding [this question](https://github.com/CliMA/CLIMAParameters.jl/pull/20#discussion_r423896240) on how to proceed.